libvast/test/range_map.cpp

```diff
@@ -15,6 +15,10 @@
 
 #include "vast/test/test.hpp"
 
+// CAF 0.17.2 added a test DSL macro named `inject`, which conflicts with
+// `range_map::inject`
+#undef inject
+
```

Comment:
For a follow-up issue: maybe we should not include the DSL unconditionally in `vast/test/test.hpp` instead? Since it's not part of the core unit test suite, we may want to include only where we actually use the DSL.

Reply:
We don't actually use the DSL anywhere. I've hidden it behind the macro `VAST_USE_TEST_DSL` for now.